ion/timestamp.go

```diff
@@ -482,8 +470,23 @@ func (ts Timestamp) String() string {
 		// This position better be right of 'T'
 		if index != -1 && tIndex < index {
 			zeros := strings.Builder{}
-			zeros.WriteByte('.')
-			for i := uint8(0); i < ts.numFractionalSeconds; i++ {
+			numTrailingZeros := 0
+
+			// Specify trailing zeros if fractional precision is less than the nanoseconds.
+			// e.g. A timestamp: 2021-05-25 13:41:31.00001234 with fractional precision: 2 will print 2021-05-25 13:41:31.00
```

Comment:
Let's use the the same format as the strings we're processing. ie `2021-05-25T13:41:31.00001234Z` rather than `2021-05-25 13:41:31.00001234`.

Reply:
Also `print` is a bit confusing since we're not actually printing anything here.

Reply:
Updated